blas/testblas/common.go

```diff
@@ -349,3 +349,62 @@ func max(a, b int) int {
 	}
 	return a
 }
+
+// packHermitian returns an n×n Hermitian matrix in packed format stored in
+// the uplo triangle of the general matrix A.
+func packHermitian(uplo blas.Uplo, n int, a []complex128, lda int) []complex128 {
```

Comment:
Can we have a round trip test for this pair of helpers?

Reply:
Done.